web/client/epics/playback.js

```diff
@@ -30,7 +32,16 @@ const BUFFER_SIZE = 20;
 const PRELOAD_BEFORE = 10;
 const domainArgs = (getState, paginationOptions = {}) => {
     // const timeData = timeDataSelector(getState()) || {};
-    return ['http://cloudsdi.geo-solutions.it:80/geoserver/gwc/service/wmts', "landsat8:B3", "time", {
+   const selectedLayerId = selectedLayerSelector(getState());
+    const layer = get(getLayerFromId(getState(), selectedLayerId)
```

Comment:
Some missing parentheses closing here?

Reply:
yes, that's my fault, I have committed a code that was meant to explain. it's this new feature that got me confused.